src/components/Map/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { BikeRentalStation, Scooter } from '@entur/sdk'
+import React, { useState } from 'react'
+
+import ReactMapGL, { Marker } from 'react-map-gl'
+
+import PositionPin from '../../assets/icons/positionPin'
+import ScooterOperatorLogo from '../../assets/icons/scooterOperatorLogo'
+
+import { DEFAULT_ZOOM } from '../../constants'
+import { useSettingsContext } from '../../settings'
+import { StopPlaceWithDepartures } from '../../types'
+
+import BikeRentalStationTag from './BikeRentalStationTag'
+import StopPlaceTag from './StopPlaceTag'
+
+import './styles.scss'
+
+const MapView = ({
+    stopPlaces,
+    bikeRentalStations,
+    scooters,
+    walkTimes,
+    interactable,
+}: Props): JSX.Element => {
+    const [settings, { setZoom }] = useSettingsContext()
+    const [viewport, setViewPort] = useState({
+        latitude: settings?.coordinates?.latitude,
+        longitude: settings?.coordinates?.longitude,
+        width: 'auto',
+        height: '100%',
+        zoom: settings?.zoom ?? DEFAULT_ZOOM,
+        maxZoom: 18,
+        minZoom: 13.5,
+    })
+    return (
+        <ReactMapGL
+            {...viewport}
+            mapboxApiAccessToken={process.env.MAPBOX_TOKEN}
+            mapStyle={process.env.MAPBOX_STYLE_MAPVIEW}
+            onViewportChange={
+                interactable
+                    ? (vp): void => {
```

Comment:
Kanskje heller kalle `vp` for `newViewport`, som er meir beskrivande?

Reply:
Endret!!